Content.Shared/Cargo/Prototypes/CargoBountyPrototype.cs

```diff
@@ -39,6 +39,12 @@ public sealed partial class CargoBountyPrototype : IPrototype
     /// </summary>
     [DataField]
     public string IdPrefix = "NT";
+
+    /// <summary>
+    /// When true, the arbitrage test skips this bounty.
+    /// </summary>
+    [DataField]
+    public bool AllowArbitrage;
 }
```

Comment:
Wouldn't it be better to just test if this is possible with cargo orders?

Reply:
the test already checks every cargo order wym

Reply:
@metalgearsloth please elaborate